pkg/proto/aksnodeconfig/v1/tlsbootstrappingconfig.proto

```diff
@@ -1,13 +1,26 @@
 syntax = "proto3";
 package aksnodeconfig.v1;
- 
+
+enum BootstrappingAuthMethod  {
+  //nolint:gosec // this is a const string to use in switch statements, not hardcoded credentials
+  UseTLSBootstrapToken = 0;
+  UseSecureTLSBootstrapping = 1;
+  UseArcMsiToMakeCSR = 2;
+  UseAzureMsiToMakeCSR = 3;
+  UseArcMsiDirectly = 4;
+  UseAzureMsiDirectly = 5;
```

Comment:
can we drop the `Use` prefix since it's not adding any parameterization / distinctive value?

Reply:
Good idea.